shared/src/helpers/date/sort-dates/sort-dates.helper.ts

```diff
@@ -0,0 +1,20 @@
+import { compareAsc, compareDesc } from 'date-fns';
+
+type sortingType = 'desc' | 'asc';
+
+interface ShouldHaveCreatedAtProp {
+  createdAt: string;
+}
+
+const sortDates = <T extends ShouldHaveCreatedAtProp>(
+  items: T[],
+  type: sortingType,
+): T[] => {
+  return items.sort((a, b) =>
+    type === 'desc'
+      ? compareDesc(new Date(a.createdAt), new Date(b.createdAt))
+      : compareAsc(new Date(a.createdAt), new Date(b.createdAt)),
+  );
```

Comment:
```suggestion
  const sortingHelper = type === 'desc' ? compareDesc : compareAsc;
 
  return items.sort(sortingHelper);
```

Reply:
Can you please explain. 
In my caase we got an array of objects - messages, and I sort it by the crearedAt field in every object.
If I sort only an array of Dates, then I also have to sort an array of messages by the sorted array of Dates?

Reply:
yes, you are right, then it should be in local helpers, not in shared